pkg/actors/placement/placement.go

```diff
@@ -142,12 +142,12 @@ func (p *actorPlacement) AddHostedActorType(actorType string) error {
 
 // Start connects placement service to register to membership and send heartbeat
 // to report the current member status periodically.
-func (p *actorPlacement) Start(ctx context.Context) error {
+func (p *actorPlacement) Start(ctx context.Context) {
```

Comment:
Let's keep the errors here so the interface can return errors. It was added recently to help with future expansion.


```suggestion
func (p *actorPlacement) Start(ctx context.Context) error {
```